karma.conf.js

```diff
@@ -0,0 +1,82 @@
+// Karma configuration
+
+module.exports = function(config) {
+  config.set({
+
+    // base path that will be used to resolve all patterns (eg. files, exclude)
+    basePath: '',
+
+    // frameworks to use
+    // available frameworks: https://npmjs.org/browse/keyword/karma-adapter
+    frameworks: ['jasmine'],
+
+    plugins: [
+      'karma-coverage',
+      'karma-chrome-launcher',
+      'karma-jasmine'
+    ],
+
+    // list of files / patterns to load in the browser
+    files: [
+      'app/public.js',
+      {pattern: 'test/**/*.png', included: false},
```

Comment:
@tabanliviu READABILITY - Double negative - why `include: false` when you can exclude?